docs/index.md

```diff
@@ -98,7 +100,7 @@ A summary of fields (details are provided below):
 | [type](#manifest-type)                                          | String  | Yes                           | Yes      | The image type. One of "zone-dataset" for a ZFS dataset used to create a new SmartOS zone, "lx-dataset" for a Lx-brand image, "lxd" for a LXD image, "zvol" for a virtual machine image or "other" for image types that serve any other specific purpose. |
 | [os](#manifest-os)                                              | String  | Yes                           | Yes      | The OS family this image provides. One of "smartos", "windows", "linux", "bsd", "illumos" or "other".                                                                                           |
 | [origin](#manifest-origin)                                      | UUID    | No                            | No       | The origin image UUID if this is an incremental image.                                                                                                                                          |
-| [files](#manifest-files)                                        | Array   | Yes (if activated)            | No       | An array with a single object describing the image file.                                                                                                                                        |
+| [files](#manifest-files)                                        | Array   | Yes (if activated)            | No       | An array with of object describing the image files.                                                                                                                                             |
```

Comment:
"array with of object"... seems like a typo.

Reply:
fixed